code/gcy/continuous/gcy_wc_ratio_continuous.py

```diff
@@ -62,8 +62,8 @@ def build_grid(gcy,
     # z' = ρ * z + ρ_π * z_π + σ_z * η0
     h_z_max = num_std_devs * jnp.sqrt(s_z**2 / (1 - ρ_z**2))
     σ_z_max = φ_z * jnp.exp(h_z_max)
-    z_max = zπ_grid[-1] + num_std_devs * σ_z_max
-    z_min = zπ_grid[0] - num_std_devs * σ_z_max
+    z_max = zπ_grid[-1] / (1 - ρ) + num_std_devs * σ_z_max
+    z_min = zπ_grid[0] / (1 - ρ) - num_std_devs * σ_z_max
```

Comment:
I think something like the following makes more sense:

```python
    z_max = (ρ_π * zπ_grid[-1] + num_std_devs * σ_z_max) / (1 - ρ)
    z_min = (ρ_π * zπ_grid[0] - num_std_devs * σ_z_max) / (1 - ρ)
```

Reply:
I am trying to test this code using
```
wc_ratio_continuous(GCY())
```
but takes too long to run and I eventually have to terminate that. Maybe because JAX isn't that compatible with Apple M1 GPU. I even tried to run that on Colab GPU but it still takes too long for me. I have been trying that for the last 2 days and so I'm unable to test the exact results.

Reply:
@JunnanZ Can you please help me debugging the exact issue? I'm trying locally but that takes too long for me. Many thanks in advance.

Reply:
Hi @Smit-create, the code is running fine, but my machine (with an RTX 3090) also struggles with the computation with a state space of moderate size, say 5^6. The results also change a lot if I change the size of the state space. Another problem is that I can't get Anderson Acceleration and Newton's method to work. They generate negative or nan values. 

I've checked your code a few times and didn't find any errors, so it's hard to say what the problem is. Maybe the state space is just too small to compute accurate results, but we are limited by our hardwares...

Reply:
Thanks, @JunnanZ. If you think this is good to merge then I can clean up some code/comments so that it's ready to go in.

Reply:
Yeah, it would be great if you could clean up the code and comments. Thanks. 

By the way, for you to test the code on your machine, maybe you can choose a smaller `d` (number of quadrature nodes) and smaller grid sizes.